components/Panel/Panel.jsx

```diff
@@ -3,20 +3,36 @@
 import React, { Component } from 'react'
 import classnames from 'classnames'
 import { isEmpty } from 'lodash'
+import FontAwesome from 'react-fontawesome'
+import Text from '../Text/Text'
 
 type Props = {
+  bodyClass?: string,
   children: Array<Component>,
   className?: string,
-  bodyClass?: string,
+  collapsed?: boolean,
+  collapsible?: boolean,
   headingLinks?: Array<Component>,
   title?: string,
+  titleIcon?: string,
+  subTitle?: string,
 }
 
 import styles from './panel.scss'
 
 export default class Panel extends Component<Props> {
   static defaultProps = {
     className: "panel-default",
+    collapsed: false,
+    collapsible: false,
+  }
+
+  state = {
+    collapsed: false,
```

Comment:
You should be able to initialize the state from the passed in props and avoid having to implement `componentWillMount` to trigger a rerender:

```js
state = {
  collapsed: this.props.collapsed,
}
```